### PR TITLE
Update wrt. 1.2.0

### PR DESCRIPTION
**Pull request by mahilleb-msft**

### CODE REVIEW

File: quickstart/python/README.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+* A subscription key for the Speech service. See [Try the speech service for free](https://docs.microsoft.com/azure/cognitive-services/speech-service/get-started).
+* Python 3.5 (64 bit) or later needs to be installed. Downloads are available [here](https://www.python.org/downloads/).
+* The Python Speech SDK package is available for Windows (x64), Mac (macOS X version 10.12 or later), and Linux (x64).
+* On Ubuntu 16.04, run the following commands for the installation of required packages:
+  ```sh
+  sudo apt-get update
+  sudo apt-get install build-essential libssl1.0.0 libcurl3 libasound2 wget
+  ```
+* On Windows you also need the [Microsoft Visual C++ Redistributable for Visual Studio 2017](https://support.microsoft.com/help/2977003/the-latest-supported-visual-c-downloads) for your platform.
```
Comment (zhouwangzw) [inline quickstart/python/README.md]: Anything need for Mac?

File: samples/js/browser/index.html [hunk 2]
```diff
@@ -413,12 +413,12 @@ <h2 style="font-weight:500;">Microsoft Cognitive Services</h2>
 
                 // Defines the language(s) that speech should be translated to.
                 // Multiple languages can be specified for text translation and will be returned in a map.
-                speechConfig.addTargetLanguage(languageTargetOptions.value.split(":")[0]);
+                speechConfig.addTargetLanguage(languageTargetOptions.value.split("(")[1].substring(0, 5));
```
Comment (zhouwangzw) [inline samples/js/browser/index.html]: Why do we need substring(0,5)?

File: samples/js/browser/index.html [hunk 3]
```diff
@@ -433,7 +433,7 @@ <h2 style="font-weight:500;">Microsoft Cognitive Services</h2>
                     window.console.log(e);
                     statusDiv.innerHTML += "(recognizing) Reason: " + SpeechSDK.ResultReason[e.result.reason] + " Text: " + e.result.text + " Translations:";
 
-                    var language = languageTargetOptions.value.split("-")[0];
+                    var language = languageTargetOptions.value.split("(")[1].substring(0, 2);
```
Comment (zhouwangzw) [inline samples/js/browser/index.html]: Are we sure that the target language returned is only 2 characters? For some languages, you can specify like zh-hans, zh-hant, or tlh-qaak.

File: samples/js/browser/index.html (hunk 2 above)
Comment (zhouwangzw) [reply samples/js/browser/index.html]: Some languages are: zh-Hans, zh-Hant, and tlh-Qaak

File: samples/js/browser/index.html [hunk 4]
```diff
@@ -449,7 +449,7 @@ <h2 style="font-weight:500;">Microsoft Cognitive Services</h2>
                     window.console.log(e);
                     statusDiv.innerHTML += "(recognized)  Reason: " + SpeechSDK.ResultReason[e.result.reason] + " Text: " + e.result.text + " Translations:";
 
-                    var language = languageTargetOptions.value.split("-")[0];
+                    var language = languageTargetOptions.value.split("(")[1].substring(0, 2);
```
Comment (zhouwangzw) [inline samples/js/browser/index.html]: same here

File: samples/js/browser/index.html [hunk 5]
```diff
@@ -583,8 +583,8 @@ <h2 style="font-weight:500;">Microsoft Cognitive Services</h2>
                 // This can happen for two broad classes or reasons.
```
Comment (zhouwangzw) [inline samples/js/browser/index.html]: Maybe we should mention that Reason is used to differentiate whether it is Error or EndOfStream

File: samples/js/browser/index.html (hunk 2 above)
Comment (zhouwangzw) [reply samples/js/browser/index.html]: These long language names are not listed in samples. Fine for now.

**Comment by mahilleb-msft** [comment]
@zhouwangzw - merged for now, as we discussed.

**Pull request merged**